rules/do-you-have-an-endomarketing-strategy-in-your-company/rule.md

```diff
@@ -1,112 +1,61 @@
-
-
-
-
-
-This strategy is mainly directed at creating an image of a great workplace and making people feel like their current position is a dream job. The prefix ‘endo’ comes from the Greek meaning "inside," therefore Endomarketing in a business environment is targeted at your employees. 
-
-
+Endomarketing (also known as **Internal Marketing**) is one of the most effective strategies to reduce your employee turnover rate and attract new, highly qualified professionals to your company. 
 
+This strategy is mainly directed at creating an image of a great workplace and making people feel like their current position is a dream job. 
 
+::: info
+The prefix ‘endo’ comes from the Greek meaning "inside," therefore Endomarketing in a business environment is targeted at your employees. 
+:::
 
 You can also call this Internal Marketing, as it seeks to improve the image of the company among its employees, culminating in a motivated team and reduced turnover.
 
-
-
-
 <!--endintro-->
 
-Many companies invest a lot of time, resources and money to attract and retain customers,  **but sometimes they completely forget about their own workers** . After implementing a new successful strategy for your internal marketing, you should see a lift in your team’s perception of the company and they will begin to see the company in a more human and charismatic way; often developing a feeling of "adoration" for the business.
-
-
-
-
-
-
-
-
-
-
-
-Richard Branson once said: " ***If you take care of your employees, they will take care of the clients,"*** and this is another reason why you should start taking care of your team in a more hands-on way. 
-   
-
-
-
-
-
-
-
-
+Many companies invest a lot of time, resources and money to attract and retain customers, **but sometimes they completely forget about their own workers**. After implementing a new successful strategy for your internal marketing, you should see a lift in your team’s perception of the company, and they will begin to see the company in a more human and charismatic way; often developing a feeling of "adoration" for the business.
 
+Richard Branson once said: *"**If you take care of your employees, they will take care of the clients**"* and this is a powerful reason why you should start taking care of your team in a more hands-on way. 
 
+According to the article [Endomarketing campaigns to generate engagement](https://race.agency/public-relations-brazil/endomarketing-campaigns/), to implement Endomarketing in a company, you must have in mind that internal marketing is not only restricted to internal "advertising", but you also need to engage your HR, with the role of providing better conditions for employees and meeting their demands. That way, proactivity will find a healthier and more balanced space to develop your customers' communication strategies and plans.
 
 The main benefits of this strategy are:
```

Comment:
Should this list also include something about free extra marketing from workers? You've mentioned the Richard Branson quote above, it would link back to that with a mention that happy, motivated and engaged workers will naturally promote their employer through their networks.